spec/forms/school_profile_form_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe SchoolProfileForm, type: :model, with_audited: true do
```

Comment:
What does the `with_audited` mean here?

Reply:
copy, paste. I'm not even sure I need it. I'll try and take it out

Reply:
Was it copied from somewhere else in this project? If we don't need it could you take it out from there too please? 🙂